dspace-api/src/main/java/org/dspace/app/suggestion/SuggestionTarget.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.suggestion;
+
+import org.dspace.content.Item;
+
+/**
+ * 
+ *
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ */
+public class SuggestionTarget {
+
+    private Item target;
+
+    private String source;
+
+    private int total;
+
+    public SuggestionTarget() {
+    }
+
+    /**
+     * Wrap a target person into a suggestion target.
+     * 
+     * @param item must be not null
+     */
+    public SuggestionTarget(Item item) {
```

Comment:
added. It could be eventually any kind of Item (i.e. at some point we could be able to create provider that will be able to suggest to import publication linked to a specific project or orgunit. In such case the project or orgunit item will become a valid target)